etc/ROS2TEST_QOS_PROFILES.xml

```diff
@@ -52,44 +53,55 @@ xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/5.3.1/rti_dds_qos
             <element>
               <name>dds.transport.UDPv4.builtin.recv_socket_buffer_size</name>
               <value>65535</value>
-            </element -->
-            
+            </element>
+
             <!-- === SHMEM Transport ====================================================================== -->
-            <!-- maximum message size >
+            <!-- maximum message size -->
             <element>
-              <name>dds.transport.shmem.builtin.received_message_count_max </name>
-              <value>512</value>
+              <name>dds.transport.shmem.builtin.parent.message_size_max </name>
```

Comment:
Modifying `received_message_count_max` and `received_message_count_max` don't modify `message_size_max`, which was till the default `65536`.
Considering that point cloud size is about `9830400`, I think that the delay issue was due to the message being fragmented a lot, even when sent over shared memory transport.

Here's info about the three parameters: https://community.rti.com/static/documentation/connext-micro/3.0.0/doc/html/usersmanual/transports/SHMEM.html#received-message-count-max.